src/js/hooks/outboundImport/useOutboundImportFiltering.js

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+
+const useOutboundImportFiltering = () => {
+  const [isFiltered, setIsFiltered] = useState(false);
+
+  return {
+    setIsFiltered,
+    isFiltered,
+  };
+};
```

Comment:
Maybe it's worth moving to this hook more logic related to filtering? Generally speaking, having a hook with just one setState is not worth creating this hook :smile: 
I think you can move to this hook actions like:
```
setIsFiltered((prev) => !prev);
```
and call it toggleFiltering
or things like that:
```
isFiltered
          ? (itemsWithErrors ?? [])
          : (itemsInOrder ?? []
```

Reply:
Other than this set state hook, lgtm